### PR TITLE
Add ES Lint

### PR DESCRIPTION
**Pull request by timgl**
## Changes

- add es lint action
-
-

## Checklist
- [x] All querysets/queries filter by Team (if applicable)
- [x] Backend tests (if applicable)

### CODE REVIEW

File: frontend/src/scenes/trends/ActionsLineGraph.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export function ActionsLineGraph({ dashboardItemId = null, filters: filtersParam
                     dashboardItemId
                         ? null
                         : point => {
-                              const { dataset, day, label } = point
+                              const { dataset, day } = point
```
Comment (timgl) [inline frontend/src/scenes/trends/ActionsLineGraph.js:28]: @mariusandra Any idea how to fix these kinds of errors, where you take some variables out to keep the rest of it in one variable? There's a couple of those errors around.

Comment (SanketDG) [reply frontend/src/scenes/trends/ActionsLineGraph.js:28]: https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings

Comment (timgl) [reply frontend/src/scenes/trends/ActionsLineGraph.js:28]: @SanketDG Great, thanks for that!

**Comment by timgl** [comment]
@mariusandra Added precommit hook.

File: package.json [hunk 2]
```diff
@@ -89,7 +89,8 @@
         }
     },
     "lint-staged": {
-        "*.{js,css,scss}": "prettier --write"
+        "*.{js,css,scss}": "prettier --write",
+        "*.js": "eslint frontend/src"
```
Comment (mariusandra) [inline package.json:93]: Hi, doesn't having `frontend/src` here mean that for every file changed, eslint will check all the files again? That might be a bit too much? Just removing `frontend/src` will make it a bit saner?

Comment (timgl) [reply package.json]: Yep good shout

File: package.json [hunk 3]
```diff
@@ -90,7 +89,7 @@
     },
     "lint-staged": {
         "*.{js,css,scss}": "prettier --write",
-        "*.js": "eslint frontend/src"
+        "*.js": "yarn run eslint"
```
Comment (mariusandra) [inline package.json:93]: I'm not sure I get this change together with removing the "eslint" script:

```
(env) Mariuss-MacBook-Pro ~/P/P/posthog [add-es-lint] (ruby 2.6.3p62) $ yarn run eslint
yarn run v1.21.1
error Command "eslint" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

image

Comment (timgl) [reply package.json]: Hm weird, that did work for me. I have fixed this though.

**Review by mariusandra — APPROVED** [review]
LGTM

**Pull request merged**